docs/README.md

```diff
@@ -242,7 +242,31 @@ Usage
 
 Upgrade process
 ---------------------
-To upgrade please reference _Installation process_ section but replace `helm install` by `helm upgrade` command
+
+###  Automated
+
+To upgrade please reference Installation process section but replace `helm install` by `helm upgrade` command
+
+See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation.
+
+### Manual
```

Comment:
Do we really need this part? If it failed we need to fix problem

Reply:
Ok